metaflow/graph.py

```diff
@@ -168,7 +169,8 @@ def __init__(self, flow):
         self._postprocess()
 
     def _create_nodes(self, flow):
-        module = __import__(flow.__module__)
+        importlib.invalidate_caches()
```

Comment:
Not sure if this will work, but it seemed germane: "This function should be called if any modules are created/installed while your program is running to guarantee all finders will notice the new module’s existence."
https://docs.python.org/3/library/importlib.html#importlib.invalidate_caches

Can remove if it doesn't work.

Reply:
This call would obviously have been more appropriate in `R.py`, but looks like it didn't work anyhow. Will revert.